app/src/main/java/org/mozilla/fenix/home/pocket/PocketCategoriesViewHolder.kt

```diff
@@ -55,12 +59,48 @@ class PocketCategoriesViewHolder(
         val categoriesSelections = components.appStore
             .observeAsComposableState { state -> state.pocketStoriesCategoriesSelections }.value
 
+        val wallpaperState = components.appStore
+            .observeAsComposableState { state -> state.wallpaperState }.value ?: WallpaperState.default
+        val wallpaperIsDefault = Wallpaper.nameIsDefault(wallpaperState.currentWallpaper.name)
+
+        val selectedBackgroundColor: Color?
+        val unselectedBackgroundColor: Color?
+        val selectedTextColor: Color?
+        val unselectedTextColor: Color?
```

Comment:
Could we preset these to default to `null` so we can remove the `else` condition below and simplify the code a little bit?

Reply:
I'll look into this